recipes/networkit/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "networkit" %}
+{% set version = "5.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "aefef17589991f6e4a04451164eb2d379c7301c9c85e787ef3ddf2f4777b922d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - git_url: 'https://github.com/kit-parco/{{name}}.git'
```

Comment:
Can you build from the PyPI source tarball instead? (See the example recipe for how to do this.)

See: http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
Done.